TweenController/TweenController.swift

```diff
@@ -25,54 +25,54 @@
 //  SOFTWARE.
 //
 
-public typealias TweenObserverBlock = (progress: Double) -> ()
+public typealias TweenObserverBlock = (_ progress: Double) -> ()
 
-public class TweenController {
+open class TweenController {
     
-    public private(set) var progress: Double = 0.0
-    private var descriptors = [TweenIntervalType]()
-    private var boundaries = [Boundary]()
+    open fileprivate(set) var progress: Double = 0.0
+    fileprivate var descriptors = [TweenIntervalType]()
+    fileprivate var boundaries = [Boundary]()
     
     //MARK: Public
     
     public init() {}
     
-    public func tween<T:Tweenable>(from: T, at progress: Double) -> TweenPromise<T> {
+    open func tween<T:Tweenable>(_ from: T, at progress: Double) -> TweenPromise<T> {
```

Comment:
We should keep the argument label here. See my rationale in a comment on `TweenPromise.swift` below.